app/translations/en.json

```diff
@@ -113,5 +113,6 @@
 	"Zoom Out": "Zoom Out",
 	"Zulip Help": "Zulip Help",
 	"keyboard shortcuts": "keyboard shortcuts",
-	"script": "script"
+	"script": "script",
```

Comment:
@RossBrunton I think you staged this file by accident. Would you mind unstaging it?